go/vt/vttablet/tabletmanager/rpc_agent.go

```diff
@@ -156,4 +156,5 @@ type RPCTM interface {
 
 	// Throttler
 	CheckThrottler(ctx context.Context, request *tabletmanagerdatapb.CheckThrottlerRequest) (*tabletmanagerdatapb.CheckThrottlerResponse, error)
+	GetThrottlerStatus(ctx context.Context, request *tabletmanagerdatapb.GetThrottlerStatusRequest) (*tabletmanagerdatapb.GetThrottlerStatusResponse, error)
```

Comment:
currently, GetThrottlerStatusRequest is empty. For RPC I understand that request object is created so that in future anything that needs to be sent down can be added.
But, this is an internal implementation so we do not require a `request` object here.

Reply:
I see your point that this is an internal implementation. Still, the past has taught us to have a Request. Otherwise even if we want to make an internal change in the future, we could run into a two-version release cycle.

Reply:
I think that is only valid for the externally exposed API.
I am not aware when it caused issues in the past.

Reply:
OK, so I tried removing this type. Creating a `rpc` function without parameters is not allowed. So you have to pass _some_ argument, potentially `google.protobuf.Empty`. I then looked around, and we never use this technique. We always pass our own `*Request` and we have multiple similarly empty `*Request` objects. So the empty `tabletmanagerdatapb.GetThrottlerStatusRequest` definition here is inline with the rest of our proto conventions.